pyxrf/model/fileio.py

```diff
@@ -88,12 +88,62 @@ def __init__(self, **kwargs):
         self.mask_data = None
 
         # Display PyXRF version in the window title
-        global pyxrf_version
-        if pyxrf_version[0] not in 'vV':
-            pyxrf_version = f"v{pyxrf_version}"
-        self.window_title_base = f"PyXRF: X-ray Fluorescence Analysis Tool ({pyxrf_version})"
+        ver, new_ver = self._get_pyxrf_version_str()
+        ver_str = ver
+        if new_ver:
+            ver_str += f" - new version {new_ver} is available"
+        self.window_title_base = f"PyXRF: X-ray Fluorescence Analysis Tool ({ver_str})"
         self.window_title = self.window_title_base
 
+    def _get_pyxrf_version_str(self):
+        """
+        The function returns the tuple of strings:
+        - current version number of PyXRF;
+        - the latest version of PyXRF from nsls2forge channel.
+        If current version is the latest, then the second string is empty
+        """
+
+        import requests
+        from distutils.version import LooseVersion
+
+        # Determine the current version of PyXRF
+        global pyxrf_version
+        pyxrf_version_str = pyxrf_version
+        if pyxrf_version_str[0] not in 'vV':
+            pyxrf_version_str = f"v{pyxrf_version_str}"
+
+        # Now find the latest version available at nsls2forge
+        pyxrf_latest_version_str = ""
```

Comment:
Will it make sense to use `None` for that if there is no new version? As you don't concatenate strings here, I think it's OK. If so, the docs above should be updated.
```suggestion
        pyxrf_latest_version_str = None
```

Reply:
You may be right. Initially this function was supposed to return complete formatted string. In current implementation I think using using None is fine.